Update asterisk on required form fields (#3948)
canonical-web-and-design/vanilla-framework: templates/docs/base/forms.md

## Required

> By applying the class `.is-required` the attribute specifies that an input field must be filled out before submitting the form. You should also add a legend to the form to be explicit about the meaning of the asterisk.

Reviewer: I've read that active voice is preferred when writing documentation so how about 
```suggestion
Apply the class `.is-required` to mandatory input fields. You should also add a legend to the form to be explicit about the meaning of the asterisk.
```

Author: Done 👍🏼
